data/partials/platforms.yaml

```
@@ -47,6 +47,10 @@ platforms:
     link: "agent/basic_agent_usage/saltstack/"
     image: "platform_saltstack.png"
     category: "configuration_management"
+  - name: foundry
+    link: "/integrations/"
```

```suggestion
    link: "https://app.datadoghq.com/fleet/install-agent/latest?platform=cloud_foundry"
```
Assuming this is the correct link based on the in-app installation guide linked for Google Cloud Run: https://docs-staging.datadoghq.com/ida.adjivon/DOCS-11593-gcp/serverless/google_cloud_run/containers/sidecar/?tab=nodejs#overview.